airflow/utils.py

```diff
@@ -403,6 +403,11 @@ def is_in(obj, l):
     return False
 
 
+def get_val_or_default( context, key, default ):
+    if key in context:
+        return context[key]
+    return default
+
```

Comment:
removed util function and changed to `dict(foo='bar').get('foo', 'default_if_key_not_found')` style.